ludwig/utils/tokenizers.py

```diff
@@ -804,7 +811,7 @@ def get_pad_token(self) -> str:
         # HACK(geoffrey): gpt2 has no pad token. Recommendation is to use eos token instead.
         # https://github.com/huggingface/transformers/issues/2630#issuecomment-1290809338
         # https://github.com/huggingface/transformers/issues/2648#issuecomment-616177044
-        if self.pretrained_model_name_or_path == "gpt2":
+        if self.pretrained_model_name_or_path in {"gpt2", "eachadea/vicuna-13b-1.1"}:
```

Comment:
Agreed!

My suggestion is to add a few new functions and call them at the end of __init__():

1. set_pad_token()
2. set_pad_token_id()

Arguably setting pad_token should set pad_token_id automatically, but my understanding is that this is not always the case (it can get messed up sometimes), so it's best to explicitly set both

Reply:
Done